### ISSUE TITLE
Visually hidden checkbox/radio buttons aren't accessible to screenreader users navigating by touch

### ISSUE THREAD

**Issue by bernardwang** [uswds/uswds#4082]
### Describe the bug
Native `<input>` elements inside checkbox/radio buttons are visually hidden using `left: -999em`, which preserves native functionally for most screenreaders. However, users navigating by touch (i.e. [Talkback's "Explore by touch" functionality](https://motorola-global-en-uk.custhelp.com/app/answers/indevice_detail/a_id/104169/p/30,6720,9299)) might not be able to access the native input because it is positioned off screen.

### Steps to reproduce the bug
1. Go to [USWDS checkbox or radio buttons](https://designsystem.digital.gov/components/form-controls/#checkbox)
2. Use "Explore by touch" in Talkback
3. Try to explore and find the native input by touching the screen with one finger

### Expected behavior
The native input is visually hidden, but positioned on top of the custom checkbox/radio icon. This can be achieved with `opacity: 0` and some additional styles to ensure its size and position matches the custom icon. 

### Additional references
- https://www.sarasoueidan.com/blog/inclusively-hiding-and-styling-checkboxes-and-radio-buttons/
- https://24ways.org/2018/inclusive-considerations-when-restyling-form-controls/

**Comment by mgifford** [comment]
Is there not a CSS class that just takes care of this all?  
https://github.com/h5bp/main.css/issues/12

I guess that there is `sr-only` 
https://github.com/uswds/uswds/blob/main/src/stylesheets/core/mixins/_screen-reader.scss

The advantage to using a class is that you can adopt your strategy if screen readers or browsers change their behavior (which they do from time to time).

**Comment by wma-rei** [comment]
Our project have ran into a related issue with the replaced checkbox and radio button controls that have caused it to fail section 508 testing in Firefox (though not Chromium-based browsers like Edge). If this should be a separate issue report let me know.

To reproduce:

1. Install ANDI from https://www.ssa.gov/accessibility/andi/help/install.html
2. Navigate to https://designsystem.digital.gov/components/checkbox/
3. Run ANDI and select the module "hidden content"
4. It reports an Alert of type "Content has been injected using CSS pseudo-elements ::before or ::after."

This also looks to be a side effect of the replaced control.

**Comment by amyleadem** [comment]
@mejiaj Are you able to confirm this using Talkback on Android devices?

**Comment by mejiaj** [comment]
@amyleadem @thisisdano I was able to use all instances of USA checkbox and radio. There's a small issue on the initial tap to focus, but it still announces checked/unchecked on the items I've tapped.

**Comment by amyleadem** [comment]
We are closing this issue because we could not reproduce. If you have questions or can provide information we are missing, please feel free to reopen this issue.
